pom.xml

```diff
@@ -88,11 +88,11 @@
     <hbase.version>1.6.0</hbase.version>
     <junit.jupiter.version>5.7.2</junit.jupiter.version>
     <kafka.version>2.6.2</kafka.version>
-    <proto.version>3.15.8</proto.version>
-    <protoc.version>3.15.8</protoc.version>
-    <grpc.version>1.37.1</grpc.version>
+    <proto.version>3.17.3</proto.version>
+    <protoc.version>3.17.3</protoc.version>
+    <grpc.version>1.40.1</grpc.version>
```

Comment:
Why not directly `1.42.1` which is latest? Also protobuf can be probably updated to `3.19.1`

Reply:
I keep the same version as Beam has. It is not (probably) required, because we shade everything, but "safetyfirst".

Reply:
ok